### DISCUSSION TITLE
Nuxt Tailwindcss color-adjust warning

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17648] [Questions]
------------------------------
- Operating System: `Linux`
- Node Version:     `v16.14.0`
- Nuxt Version:     `3.0.0-rc.1-27520989.22954d0`
- Package Manager:  `yarn@1.22.18`
- Builder:          `vite`
- User Config:      `meta`, `runtimeConfig`, `buildModules`, `tailwindcss`, `vite`, `css`, `router`, `nitro`, `srcDir`, `build`, `modern`
- Runtime Modules:  `-`
- Build Modules:    `@nuxtjs/tailwindcss@5.0.3`, `@vueuse/nuxt@8.3.1`
------------------------------
Today after update my nuxt 3 verison to last - i have an warning from tailwindcss:
![image](https://user-images.githubusercontent.com/25690524/166198738-00eeef4f-791a-4a92-89b1-149f22b9e6c7.png)
Can you tell me please how to resolve it?

**Comment by ayavrik** [comment]
I ran into the same issue. Fixed it by installing autoprefixer version 10.4.5
`npm install autoprefixer@10.4.5`

**Reply by mihail727** [reply]
shouldn't autoprefixer be part of the `@nuxt/tailwindcss` package?
I just think that this is a defect of this particular module

**Reply by mihail727** [reply]
I created an issue in their [repository](https://github.com/nuxt-community/tailwindcss-module/issues/466), I hope they will fix it soon, otherwise why put an extra package
